lib/ansible/modules/cloud/openstack/os_server_facts.py

```diff
@@ -47,6 +47,13 @@
           of additional API calls.
      required: false
      default: false
+   filters:
+     description:
+        - restrict results to servers matching a dictionary of
+          filters
+     required: false
+     default: None
+     version_added: "2.4"
```

Comment:
Sorry for the delay, could you bump this to `version_added: "2.8"` and we'll be able to get this merged.